### ISSUE TITLE
Update Sandbox Filenames

### ISSUE THREAD

**Issue by GentrySteven** [DLFMetadataAssessment/Sandbox#200]
**To be completed once other work on the sandbox has been done!**

Use the below information to update the **human-created (e.g. markdown)** filenames in this repository and the links in the repository's markdown pages. 

Although members of the Documentation sub-group (Staci, Gabi, and Steven) can work on this project, we would like to invite other members of this sub-group to participate in this project and get more comfortable with Git and GitHub! Let us know if you want to participate!

Also, once we begin work on this project, we will probably split this issue into several, smaller issues for members to complete.

Assignment of responsibilities:

- [x] Review and update the filenames of all the human-created files in the Sandbox.
- [x]  Review and update the links in markdown files (**either all the files or a certain number - your choice!**): Done (Steven G.).
- [x]  Review the updated website to ensure nothing is broken and the links point to the correct pages.

Guidelines:

- Try to use create descriptive, concise file names.
- Enter new files in lowercase (i.e. avoid camel case). Separate multiple words using hyphens (e.g., revision-file.txt)
- Except for hyphens, do not use special characters in file names (.\/:*?“<>|[]&$,). Use underscores in limited circumstances
- Limit file names to 30 characters or fewer. 
- If file name contains dates, format in ISO 8601 (YYYY-MM-DD or YYYYMMDD, e.g.). Make sure date format conforms to others in the directory.
- Ensure file name ends with a period followed by a file extension (.txt, .jpg, etc.). **For markdown files, the file extension used will be .md**
- If beginning a file name using a number, use leading zeroes to ensure subsequent files sort correctly. 
- Avoid using articles (a, an, the) and common conjunctions/prepositions (and, of, to, etc.) in file names, unless they are integral to understanding the file.

General Workflow:

1. Review and update the files in the **entries**, **img**, and possibly **asset** directories. Feel free to suggest others in the comments below!
2. Review and update links in various markdown files.
3. Check to make sure nothing has been broken and the links work as intended. 
4. Repeat as necessary.

Example:

After reviewing the **entries** directory, someone changed the **project_archived.md** file to **project-archived.md** (i.e. removed the underscore) and **projects.markdown** to **projects.md** (i.e. changed the file extension to .md). 

Once this was done, someone would then review the appropriate markdown files and update the links as necessary.

**NOTE**

See comments below for additional information/changes.

**Comment by GentrySteven** [comment]
Updating filenames in clone - done.

**Comment by GentrySteven** [comment]
And apologies. I was working too quickly and accidentally merged something into the main branch of this repository. It shouldn't matter too much, as I both reversed the changes using another pull request...and the same pull request should be opened shortly. 

But if you see anything weird, please feel free to correct it!

**Comment by GentrySteven** [comment]
Hi @kateefly! I have good news and bad news. 

The good news is that I have made modifications to all the filenames in the repository, as well as fixed the bulk of the issues mentioned in the comment above. See [this link](https://github.com/DLFMetadataAssessment/Sandbox/commit/4ce39c36bdff7e79e993c4b4f5e01dd3d68b9d7e) for a somewhat messy listing of all relevant changes. 

The only thing left to do will be to change--on the "Metadata Assessment Framework and Guidance" page--the link in the sentence "Please see our list of recommended publications for further investigation" (I believe it currently is "Sandbox/publications" as opposed to "/Sandbox/publications".

The bad news is that, in my haste, I accidentally merged the branch containing our commits into the main of THIS repository (as opposed to the main branch of my repository). I sucessfully reverted my pull request, but--when you compare the main branch of my fork with the main branch of this repository--no changed files are listed. I assume this is because the commits are in both my fork and this repository.

I was debating how to remove those commits, but figured I'd reach out to you and see if you had any suggestions before I began mucking around too much more with our main sandbox repository. 

Worse case scenario, do you think we could just--if you approve of my changes--reverse the pull request revert I made? I feel like that could fix things...

In any event, I'm super sorry about this! Blame it on my eagerness to knock this project out before the work week began...

**Comment by kateefly** [comment]
Hi @GentrySteven, from what I've been able to tell, I think part of the issue is that the original branch with the changes (issue#200-round-3) was deleted. Do you still have this branch on your local machine (and can you push it up)? If so, I think you can create a new pull request and have me approve it.

Otherwise, I think the best course is to revert the reverted changes, as you mentioned. But I'd like to try the other way first, if possible.

**Comment by GentrySteven** [comment]
Hi @kateefly - 

Thankfully, I did have that local branch on my laptop was able to push it up to my fork (see [here](https://github.com/GentrySteven/Sandbox/tree/issue%23200-round-3)). Unfortunately, when I went to create a new pull request for you to review, I got this message: 

![image](https://user-images.githubusercontent.com/49626375/134269393-1afd5ad5-df48-4fd8-8276-5c903c609882.png)

As such, perhaps it would be best to revert the revert? The only other thing I found in the Git documentation was re: git re-basing/resetting, but--at least from what I could in the documentation--that approach was something to be avoided if at all possible.

In any event, I'll also make sure the test documentation site has some instructions re: how to address this issue if it comes up again.

**Comment by kateefly** [comment]
Hm..when I tried to revert it, I wasn't able to do so. It gave me this message:

![image](https://user-images.githubusercontent.com/15897192/134393857-4977ebaf-223b-4aac-955b-e1959b363592.png)

I'm trying now to think of the best troubleshooting/next steps options. Could you try pull changes on your branch and then seeing if you're able to do a pull request then? 

We may also be able to do some troubleshooting tomorrow at the group meeting.

**Comment by GentrySteven** [comment]
Ack. I bet that issue is=s due to Shaun's recent merging of his pull request from a few days ago. It still doesn't look like I can do the pull request. 

Unfortunately, I'm not going to be at tomorrow's meeting as I'll be at the Dentist's.

Hrm. Let me try something...

**Comment by GentrySteven** [comment]
Ok @kateefly - so I just now tried something on [a branch on my fork](https://github.com/GentrySteven/Sandbox/commit/4b314d43e76d50bcf9efe4518d6323e2fdb01067). 

Basically, I manually added some of the updated files from my local repo (which haven't been changed since I made the original request about 10 days ago) to a new branch and tried to merge this branch into another test branch. This time, it seemed like the pull request could finally go through.

I'll try to create a new pull request this way and will ping you when it's good to go. If you don't forsee or encounter any issues, we could try accepting that pull request (and hope our issue has finally been solved.)
